packages/ml_api/requirements/requirements.txt

```diff
@@ -1,12 +1,13 @@
 # ML Model
-tid-gradient-boosting-model>=0.1.18,<0.2.0
+tid-gradient-boosting-model>=0.3.0,<0.4.0
```

Comment:
Key fix for section 9 (2/2)

Note required new publishing of gradient boosting model (so sklearn versions are compatible)